tools/regenByteCode.py

```diff
@@ -29,58 +31,93 @@ def run_sub(message, commands, error):
     if sub.returncode != 0:
         sys.exit(error)
 
+
 if len(sys.argv) == 1 or sys.argv[1] != '--skip-build':
     run_sub('Compiling ChakraCore with no Jit',
-        ['../build.sh', '--no-jit', '--test-build', '--target-path=../out/noJit', '-j=2'], 
+        ['../build.sh', '--no-jit', '--test-build', '--target-path=../out/noJit', '-j=8'], 
         'No Jit build failed - aborting bytecode generation')
     run_sub('Compiling ChakraCore with Jit',
-        ['../build.sh', '--test-build', '--target-path=../out/Jit', '-j=2'], 
+        ['../build.sh', '--test-build', '--target-path=../out/Jit', '-j=8'], 
         'Jit build failed - aborting bytecode generation')
 
-# Regenerate the bytecode
-def bytecode_job(outPath, command, error):
-    header = open(outPath, 'w')
-    job = subprocess.Popen(command, stdout=header)
-    job.wait()
-    if job.returncode != 0:
-        sys.exit(error)
 
-# INTL
-print('Generating INTL bytecode')
-bytecode_job('../lib/Runtime/Library/InJavascript/Intl.js.nojit.bc.64b.h',
-    ['../out/noJit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 64bit noJit bytecode')
 
-bytecode_job('../lib/Runtime/Library/InJavascript/Intl.js.nojit.bc.32b.h',
-    ['../out/noJit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '-Force32BitByteCode','../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 32bit noJit bytecode')
+# this header text will be placed at the top of the generated files
+header_text = '''//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
 
-bytecode_job('../lib/Runtime/Library/InJavascript/Intl.js.bc.64b.h',
-    ['../out/Jit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 64bit bytecode')
+// Generated Bytecode Header, this file was created by tools/regenByteCode.py
+// This file contains:
+// a) bytecode for Intl library methods implemented in javascript and
+// b) bytecode for other Js library methods, JsBuiltIns, implemented in javascript
 
-bytecode_job('../lib/Runtime/Library/InJavascript/Intl.js.bc.32b.h',
-    ['../out/Jit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '-Force32BitByteCode','../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 32bit bytecode')
+#define JsBuiltIns(VALUE)'''
```

Comment:
Set out the top of the header including beginning the JsBuiltIns #define that will be used to store the list of input files below.